crates/fuel-core/src/state/rocks_db.rs

```diff
@@ -126,13 +137,22 @@ where
     Description: DatabaseDescription,
 {
     pub fn default_open_temp(capacity: Option<usize>) -> DatabaseResult<Self> {
+        Self::default_open_temp_with_params(capacity, 512, Default::default())
+    }
+
+    pub fn default_open_temp_with_params(
```

Comment:
Maybe it makes sense to add `DatabaseConfig` or something like that to avoid adding new arguments in the future

Reply:
Done